blogposts/2021/release-3.35.md

```diff
@@ -0,0 +1,46 @@
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.35/sourcegraph-3-35-release.png
+changelogItems:
+  - description: Individual batch changes can publish multiple changesets to the same repository by specifying multiple target branches using the `on.branches` attribute.
+    url: https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on-repository
+    category: Batch Changes
+  - description: The search bar is no longer auto-focused when navigating between files. This allows you to utilize the keyboard shortcut Cmd+LeftArrow (or Ctrl-LeftArrow) to go back to the browser's previous page instead of moving the cursor to the first position of the search bar.
+    url: https://github.com/sourcegraph/sourcegraph/pull/28943
+    category: Search
+  - description: 'Using `select:repo` in search queries will now stream results incrementally, greatly improving speed and reducing time-to-first-result.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/28920
+    category: Search
+  - description: External services will now stop syncing if they exceed the user or site level limit for total number of repositories added. They will only continue syncing if the extra repositories are removed or the corresponding limit is increased.
+    url: https://github.com/sourcegraph/sourcegraph/pull/28674
+    category: Repositories
+  - description: Non-bare repositories found on gitserver will now be removed by a janitor job.
+    url: https://github.com/sourcegraph/sourcegraph/pull/28895
+    category: Repositories
+  - description: Fixed a bug where moving a changeset from draft state into published state did not work on GitLab code hosts.
+    url: https://github.com/sourcegraph/sourcegraph/pull/28239
+    category: Batch Changes
+  - description: Fixed a bug where modifying changesets on Bitbucket Server could previously fail if the local copy in Batch Changes was out of date.
+    url: https://github.com/sourcegraph/sourcegraph/pull/29100
+    category: Batch Changes
+  - description: Fixed a bug introduced in 3.34 where certain regexp syntax for repository searches caused the entire search, including non-repository searches, to fail.
+    url: https://github.com/sourcegraph/sourcegraph/pull/28826
+    category: Search
+---
+
+Sourcegraph 3.35 is now available! Here are some highlights from this release:
+
+## Publish changesets to multiple branches in a repository
+
+Batch Changes now allow you to publish changesets to [several branches in the same repository](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on-repository), rather than a single branch per repository. This is particularly useful, for example, when long-lived branches are used to define versions or releases. Bumping up a dependency or changing boilerplate in several branches of the same repository is painful, repetitive work that's a great candidate for a batch change.
```

Comment:
```suggestion
Bumping up a dependency or changing boilerplate in several branches of the same repository is painful, repetitive work that's a great candidate for a batch change. Batch Changes now allows you to publish changesets to [several branches in the same repository](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on-repository), rather than a single branch per repository. This is particularly useful, for example, when long-lived branches are used to define versions or releases. 
```